.github/workflows/ready_for_review.yml

```diff
@@ -83,33 +86,37 @@ jobs:
     steps:
       - name: Check Backend Requirement
         id: check_backend_requirement
-        if: ${{ contains(fromJSON(env.pr_requested_teams), 'backend-review-group') }}
+        if: |
+          ${{ contains(fromJSON(env.pr_requested_teams), 'backend-review-group') }} ||
+          ${{ contains(fromJSON(env.pr_labels), 'require-backend-approval') }}
```

Comment:
Check for the require-backend-approval or the backend-review-group. Found an edge case when there's a previous approval and user puts the pull request back in draft mode.